spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosDataYamlParser.java

```diff
@@ -32,9 +40,63 @@ public NacosDataYamlParser() {
 
 	@Override
 	protected Properties doParse(String data) {
```

Comment:
It is recommended to replace Proterties with LinkedHashMap , unnecessary conversion of data types is reduced , so you need to modify the affected code of data flow . Other parsers need to do the same .

Reply:
> It is recommended to replace `Proterties ` with `LinkedHashMap `, unnecessary conversion of data types is reduced , so you need to modify the affected code of data flow . Other parsers need to do the same .

this PR only for fix orderly properties not for refactor

Reply:
i hope you can do some refactors, don't want to merge some bad codes. thanks